dracut/anaconda-lib.sh

```diff
@@ -434,3 +434,14 @@ wait_for_updates() {
 wait_for_dd() {
     echo "[ -e /tmp/dd.done ]" > "$hookdir/initqueue/finished/dd.sh"
 }
+
+wait_for_disks() {
+    # Allow up to 'inst.wait_for_disks' seconds for disks to be enumerated and
+    # related udev rules to execute (defaults to 5 seconds, 0 disables the
+    # feature)
+    finished_hook="$hookdir/initqueue/finished/wait_for_disks.sh"
+    [ -e "$finished_hook" ] && return
+    DISKS_WAIT_DELAY=$(getargnum 5 0 10000 inst.wait_for_disks)
+    timeout=$(($(awk -F '.' '{print $1}' /proc/uptime) + $DISKS_WAIT_DELAY))
```

Comment:
Is it guaranteed that decimal point will always be part of `/proc/uptime`? I wonder if there are platforms providing uptime as a whole number only.

Reply:
Actually all this is quite complicated, we may rely on `main_loop` instead